theme/ckeditor5-block-quote/blockquote.css

```diff
@@ -0,0 +1,11 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+blockquote {
+	border-left: solid 5px #CCC;
```

Comment:
Like in `code.css` we can use variable here too:  `--ck-color-blockquote-border`.